src/graphql/root/mutation/user-quiz-question-update-completed.ts

```diff
@@ -16,34 +21,41 @@ const UserQuizQuestionUpdateCompletedMutation = GT.Field({
   args: {
     input: { type: GT.NonNull(UserQuizQuestionUpdateCompletedInput) },
   },
-  resolve: async (_, args, { uid, logger }) => {
-    const { id } = args.input
+  resolve: async (_, args, { uid, domainUser, logger }) =>
```

Comment:
you can get the correct type until @samerbuna PR https://github.com/GaloyMoney/galoy/pull/891 is being merged with: `{ uid, domainUser, logger }: {uid: UserId, domainUser: User, logger: Logger}`